src/applications/appeals/10182/tests/utils/focus.unit.spec.js

```diff
@@ -4,7 +4,7 @@ import { render, waitFor } from '@testing-library/react';
 
 import { $ } from 'platform/forms-system/src/js/utilities/ui';
 
-import { focusIssue } from '../../utils/focus';
+import { focusIssue } from '../../../shared/utils/focus';
```

Comment:
Sorry, I meant we could remove this entire test since it already exists in the shared test folder